hudi-common/src/main/java/org/apache/hudi/common/model/HoodieColumnRangeMetadata.java

```diff
@@ -31,15 +29,13 @@
   private final T minValue;
   private final T maxValue;
   private final long numNulls;
-  private final PrimitiveStringifier stringifier;
 
-  public HoodieColumnRangeMetadata(final String filePath, final String columnName, final T minValue, final T maxValue, final long numNulls, final PrimitiveStringifier stringifier) {
+  public HoodieColumnRangeMetadata(final String filePath, final String columnName, final T minValue, final T maxValue, final long numNulls) {
```

Comment:
Each data type has its own Stringifier type. A common one is sufficient here?

Reply:
Not sure i follow. Am actually removing it since it ain't used anywhere

Reply:
I misread. We are good here.